requirements.txt

```diff
@@ -1,6 +1,5 @@
 Flask==2.2.2
```

Comment:
@ajitava-git Also sklearn is being removed. We no longer need it?

Reply:
Yes @avinash010 . sklearn was obsolete.